public/PrimaryImage.jsx

```diff
@@ -33,25 +34,32 @@ class PrimaryImage extends React.Component {
     this.setState({
       mouseOver: false
     })
-    var el = document.getElementById('primary-image');
-    if (el) {
-      el.style.transform = 'scale(1)';
-      el.style.top = '0px';
-      el.style.left = '0px';
-    }
   }
 
   mouseExitContainer(){
-    document.getElementById('next-image-button').style.opacity = 0;
-    document.getElementById('last-image-button').style.opacity = 0;
+    this.setState({
+      mouseHovered: false
+    })
   }
 
   mouseEnterContainer(){
-    document.getElementById('next-image-button').style.opacity = 1;
-    document.getElementById('last-image-button').style.opacity = 1;
+    this.setState({
+      mouseHovered: true
+    })
   }
 
   render() {
+    const imageBtnStyle = {
+      opacity: Number(this.state.mouseHovered)
+    };
+
+    let primaryImgStyle = {}
```

Comment:
This should be const unless you're reassigning it somewhere

Reply:
Const for an object even if you mutate the properties?